### PR TITLE
PYPI release preparation

### PR DESCRIPTION
**Pull request by ktsitsi**
This PR:

- Renames the python package from `tiledbimg` to `tiledb.bioimg`
- Adds a `workflows/release.yml` job for release in testpypi and pypi
- Changes the release name from `tiledbimg` to `tiledb-bioimg` as voted by partners 

Requirements:

- We need to create Secret tokens for release under this repo for the `release.yml` action to work

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+name = tiledb-bioimg
 description = Package supports all bio-imaging functionality provided by TileDB
 author = TileDB, Inc.
 author_email = help@tiledb.io
 long_description = file: README.md
 long_description_content_type = text/markdown
 license = MIT
 license_file = LICENSE
-keywords = tiledb, tiledbimg , bioimaging
+keywords = tiledb, tiledb-bioimg , bioimaging
```
Comment (gsakkis) [inline setup.cfg:10]: `tiledb-bioimg` doesn't make much sense as a keyword

File: setup.py [hunk 2]
```diff
@@ -15,7 +15,7 @@
     use_scm_version={
         "version_scheme": "guess-next-dev",
         "local_scheme": "dirty-tag",
-        "write_to": "tiledbimg/version.py",
+        "write_to": "tiledb/version.py",
```
Comment (gsakkis) [inline setup.py:18]: Is it `tiledb/version.py` or `tiledb/bioimg/version.py` mentioned in gitignore?

**Review by gsakkis — APPROVED** [review]
LGTM :+1:

File: setup.cfg (hunk 1 above)
Comment (ktsitsi) [reply setup.cfg:10]: Borrowed it from `ML`. Can remove

File: setup.py (hunk 2 above)
Comment (ktsitsi) [reply setup.py:18]: Nice catch!

**Pull request merged**